game-runner/src/main/java/za/co/entelect/challenge/engine/runner/GameEngineRunner.java

```diff
@@ -202,14 +202,46 @@ private void publishGameComplete(boolean matchSuccessful) throws Exception {
 
         gameResult.roundsPlayed = gameMap.getCurrentRound();
         gameResult.isComplete = true;
-        gameResult.isSuccessful = matchSuccessful;
+        gameResult.isSuccessful = matchDidNotTimeout;
 
-        gameCompleteHandler.accept(gameMap, players, matchSuccessful);
+        gameCompleteHandler.accept(gameMap, players, matchDidNotTimeout);
 
-        if (!matchSuccessful) {
+        if (!matchDidNotTimeout) {
             gameResult.verificationRequired = true;
-            throw new MatchFailedException("Match Failed");
+            throw new MatchFailedException("Match timed out");
         }
+
+        int minExpectRounds = 36;
+        if (gameResult.roundsPlayed < minExpectRounds) {
+            gameResult.verificationRequired = true;
+            throw new MatchFailedException("Match duration was " + gameResult.roundsPlayed +
+                    " rounds, still less than the expected " + minExpectRounds + " rounds");
+        }
+
+        int minExpectScore = gameResult.roundsPlayed * GameConfig.getRoundIncomeEnergy() * GameConfig.getEnergyScoreMultiplier();
+        if (gameResult.playerOnePoints <= minExpectScore) {
+            gameResult.verificationRequired = true;
+            throw new MatchFailedException("Player One scored only" + gameResult.playerOnePoints +
+                    " points, not even the expected " + minExpectScore + " points");
+        }
+        if (gameResult.playerTwoPoints <= minExpectScore) {
+            gameResult.verificationRequired = true;
+            throw new MatchFailedException("Player One scored only" + gameResult.playerTwoPoints +
```

Comment:
Should this not be "Player Two scored only"?

Reply:
Thats correct. 
Fixed